zmon_aws_agent/main.py

```diff
@@ -147,6 +148,7 @@ def main():
         dynamodbs = aws.get_dynamodb_tables(region, infrastructure_account)
         certificates = aws.get_certificates(region, infrastructure_account)
         aws_limits = aws.get_limits(region, infrastructure_account, apps, elbs)
+        sqs = aws.get_sqs_queues(region, infrastructure_account)
```

Comment:
I fear, when leaving a list of entities empty and only run the discovery once in a while, the existing entities will be removed from Zmon.